Tests/Recurly/ShippingMethod_Test.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+class Recurly_ShippingMethodTest extends Recurly_TestCase
+{
+  function defaultResponses() {
+    return array(
+      array('GET', '/shipping_methods/fedex_ground', 'shipping_methods/show-200.xml'),
+    );
+  }
+
+  public function testPerformanceObligation() {
+    $shipping_method = Recurly_ShippingMethod::get('fedex_ground', $this->client);
+
+    $this->assertInstanceOf('Recurly_ShippingMethod', $shipping_method);
+    $this->assertEquals('fedex_ground', $shipping_method->code);
+    $this->assertEquals('FedEx Ground', $shipping_method->name);
+    $this->assertEquals('twywqfr48v9l', $shipping_method->liability_gl_account_id);
```

Comment:
can we also add `accounting_code` and `tax_code`. 
It would also be nice to test shipping method list (verify the count is right)

Reply:
Added those attributes although it is a tech debt from previous devs, not adding the list since is not required for this work.